### ISSUE TITLE
Dotnet Driver Urgent Bug - .NET/C# Invalid Cast

### ISSUE THREAD

**Issue by ronjonesjr** [appium/appium-dotnet-driver#230]
## Description

It seems v 3.12.0 of the Selenium.WebDriver package has a bug in it that v 3.11.2 didn't have. I have test code that locates buttons on a mobile application via AccessibilityId in Appium that works great in 3.11.2 but fails miserably in 3.12.0. It's a Windows 10 application that is being tested via Appium's included WinAppDriver.

It looks like some type must have changed in the underlying system as the exception I get with 3.12.0 is:

System.InvalidCastException: Unable to cast OpenQA.Selenium.Remote.RemoteWebElement to IFindByAccessibilityId nor to IFindsByFluentSelector

The stack trace is:
MobileBy.FindElement(ISearchContext context) <<-Exception is tossed here.
RemoteWebElement.FindElement(By by)

## Environment

Appium Desktop 1.5.0 - I'd love to update but the latest desktop has bugs.
Windows 10

**Comment by xaaviair** [comment]
Hello @ronjonesjr did you find a fix for this issue? I am using Appium 1.6.1, not sure if this is the issue. Thanks!

**Comment by ronjonesjr** [comment]
I haven't tried recently.  I'm running 1.5.0 and getting work done.  I'm hoping the owners will fix and update it before I try again.

**Comment by ronjonesjr** [comment]
Is anyone working on this?  It's a fairly big issue and preventing my C# tests from running.  Appium desktop 1.6.2 with Appium v1.8.1 still exhibits this and other invalid cast issues.

**Comment by ronjonesjr** [comment]
[SeleniumHQ/selenium#6045](https://github.com/SeleniumHQ/selenium/issues/6045) .  The Selenium team mentioned that this is a problem with the Windows/C# Appium driver caused by a Selenium change that was requested by the .NET team.  When can we expect these breaking invalid casts to be fixed?

**Comment by ronjonesjr** [comment]
Changed title.  Is anyone looking at these bugs?  This bug is breaking already running code with no workarounds.  It's making the automation test code totally useless with the WinAppDriver and C#.

**Comment by Arun-CB** [comment]
Facing same issue.

**Comment by akinsolb** [comment]
Next beta release will support Selenium v3.14. There won't be support for Selenium > v3.9 and < v3.14
